pagopa/io-app
# feat(Bonus Pagamenti Digitali): [#177161766] Add orchestration workunit for privative card

## Short description
This PR add the workunit for the onboarding of a privative card.

## List of changes proposed in this pull request
- Added the workunit for the onboarding of a privative card
- Added the saga that manage the workunit and the bpdactivation


https://user-images.githubusercontent.com/11773070/109984075-b893cd00-7d03-11eb-9f9d-1b474a24adb7.mov


## How to test
- Dispatch `walletAddPrivativeStart` to start the saga
- Dispatch `walletAddPrivativeBack`, `walletAddPrivativeCancel` or `walletAddPrivativeCompleted` to see the different behaviour
